lite-pm/pages/components/AppPage.js

```diff
@@ -183,18 +469,37 @@ class AppPage extends Component {
               projectLink={this.state.project.projectId}
             />
             <h2 className={styles.h2}>The Team</h2>
-            <div className="d-flex">{this.state.teamMembers}</div>
+            {/* add map for team members */}
+            <div className="d-flex">
+              {this.state.project.Member.map((member) => {
+                return(
+                <TeamMember
+                  key={uuid()}
+                  name={member.name}
+                  email={member.email}
+                  git={member.github}
+                  phone={member.phone}
+                ></TeamMember>);
```

Comment:
The function inside `map` can take a second parameter (index), e.g. `Member.map((member, index) => {`. You can use that index as key, e.g. `key={index}` so you don't have to generate a whole new uuid just for that